docs/isoSPI.md

```diff
@@ -1,3 +1,109 @@
-# AMDC isoSPI Interface
+# AMDC isoSPI and Differential I/O Interface
 
-This document describes the design considerations and implementation details for the isoSPI interface subsystem on the Advanced Motor Drive Controller (AMDC).
+This document describes the design considerations and implementation details for the isoSPI and differential I/O interface on the Advanced Motor Drive Controller (AMDC). The block diagram of the implementation is presented and each block is discussed in detail. Details on the PCB layout, and connector pinout are also provided in this document.
+
+
+## Relevant Versions of AMDC Hardware
+
+AMDC REV D
+
+
+## Design Requirements and Considerations
+
+The isolated serial peripheral interface (isoSPI) has increased noise immunity and safety. The following describes the design requirements for the isoSPI and differential I/O (D_IO) interface. The design requirements for the isoSPI and differential I/O (D_IO) interface are as follows:
+
+1. Implement four isoSPI and four differential I/O interfaces.
+
+2. Implement a voltage level translator to interface with the FPGA that operates at 1.8V.
+
+
+## Block Diagram / External Connections
+
+<img src="images/amdc-isoSPI.svg" />

```

Comment:
It doesn't look like you have implemented changes to the block diagram yet. Here is what I see when I view your PR:

<img width="998" alt="image" src="https://user-images.githubusercontent.com/7421348/86164691-c5f31a80-bad7-11ea-9ed4-766059f60838.png">

Reply:
The block diagram is updated